crates/pallets/starknet/src/genesis_loader.rs

```diff
@@ -50,7 +55,16 @@ impl<T: crate::Config> From<GenesisLoader> for GenesisConfig<T> {
                 let hash = unsafe { std::mem::transmute::<ClassHash, ClassHashWrapper>(hash) };
                 match class {
                     ContractClass::Path { path, version } => {
-                        (hash, get_contract_class(&read_file_to_string(&path), version))
+                        let contract_path = if loader.madara_path.is_some() {
```

Comment:
If-let pattern

Reply:
Can u give an e.g. on how to use the if-let in this scenario?

Reply:
@d-roak I think he meant 
`if let Some(contract_path) = loader.madara_path {
   ...
}`

Reply:
yep, but that way u won't get the `contract_path` defined outside the `if`

Reply:
in the else branch? 

```

if let Some(contract_path) = loader.madara_path {
    foo();
} else {
    bar();
}
```

Reply:
nop, outside. The if else is to attribute a value to contract_path (conditional value), so I can use it after that 
```
let contract_path = if loader.madara_path.is_some() {
    "foo"
} else {
    "bar"
}

quack(madara_path);
```

Reply:
true, then let's leave it as is

Reply:
```rust
let contract_path = if let Some(contract_path) = loader.madara_path {
    contract_path + "/" + &path
} else {
      let project_path = utils::get_project_path().expect("Project path not found");
       project_path + "/" + &path
};
```